dataprofiler/profilers/json_encoder.py

```diff
@@ -26,7 +26,7 @@ def default(self, to_serialize):
                 numerical_column_stats.NumericStatsMixin,
```

Comment:
Thinking on this: ` numerical_column_stats.NumericStatsMixin` would be included with any profile. Do we need it here specifically? does that cause issue? 

Not saying it is a problem, just considering the value of it here.

Reply:
This is needed only because I have a test case that serializes a NumericStatsMin `test_numeric_stats_mixin_profile.TestNumericStatsMixin.test_json_encode`. Otherwise, it's not really necessary